doc/2/guides/getting-started/run-kuzzle/index.md

```diff
@@ -115,6 +115,26 @@ When asked for credentials, just choose `Login as Anonymous`.
 
 You are now connected to your local Kuzzle application with the Admin Console! Everything is empty but we are gonna change that in the next section.
 
+::: info
+The minimum rights required for an user to connect to the Kuzzle Admin Console are:
```

Comment:
```suggestion
The minimum rights required by an user to connect to the Kuzzle Admin Console are:
```

Reply:
https://lingohelp.me/preposition-after-adjective/required-for-or-required-of-or-required-by/